modules/arm_plugin/docs/arm_opset.md

```diff
@@ -197,6 +196,7 @@ The semantics match corresponding nGraph operation classes declared in `namespac
 * `'Less'` doesn't support `broadcast` for inputs.
 * `'LessEqual'` doesn't support `broadcast` for inputs.
 * `'LRN'` layer is supported `axes = {1}` or `axes = {2, 3}` only.
+* `'MaxPool-1'` layer is supported via arm_compute library for 4D input tensor and via reference implementation for another cases.
```

Comment:
Why `MaxPool-1' used this if you support MaxPool-8?

Reply:
Both versions of MaxPool are supported. MaxPool-8 is supported via reference implementation while one case of MaxPool-1 operation (4D input tensor) is supported via ACL.